extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -341,6 +352,11 @@ export class AmpLiveList extends AMP.BaseElement {
     if (shouldSendAmpDomUpdateEvent) {
       promise = promise.then(() => {
         this.sendAmpDomUpdateEvent_();
+        const event = createCustomEvent(
+          this.win,
+          'amp-live-list.update',
+          {insertedItemIds: insertItemIds, replacedItemIds: replaceItemIds});
```

Comment:
Do we need to expose these? I can't immediately think of a good use case for this data and I'd prefer if we kept the API simpler if possible.